formats/iso0.go

```diff
@@ -127,3 +163,17 @@ func (i *ISO0) Decode(pinBlock, account string) (string, error) {
 
 	return pin, nil
 }
+
+func randomHexLetters(max int) string {
+	b := make([]byte, max)
+	n, err := io.ReadAtLeast(rand.Reader, b, max)
+	if n != max {
```

Comment:
Is there a case where n > max could be true? 

```
If min is greater than the length of buf, ReadAtLeast returns ErrShortBuffer.
On return, n >= min if and only if err == nil. 
```

Reply:
Also should this function be shared across files? I see it's reimplemented in iso1.go